benchflow-dsl/src/main/scala/cloud/benchflow/dsl/BenchFlowTestAPI.scala

```diff
@@ -50,11 +59,86 @@ object BenchFlowTestAPI {
 
   }
 
-  def validateTest(benchFlowTest: BenchFlowTest): Boolean = {
+  private val userMissing = "Missing user specification. "
```

Comment:
Move semantic validation to a different file, in a package where it is clear the responsibility of the code

Reply:
updated in new commit